shenyu-client/shenyu-client-dubbo/shenyu-client-dubbo-common/pom.xml

```diff
@@ -31,5 +31,10 @@
             <artifactId>shenyu-client-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.alibaba</groupId>
+            <artifactId>dubbo</artifactId>
+            <version>${alibaba.dubbo.version}</version>
+        </dependency>
     </dependencies>
-</project>
\ No newline at end of file
+</project>
```

Comment:
should not add `alibaba-dubbo dependency` to `shenyu-client/shenyu-client-dubbo/shenyu-client-dubbo-common/pom.xml`.

Reply:
Done